test/note_test.rb

```diff
@@ -5,16 +5,56 @@ def setup
     @frequency   = 220.to_f
     @duration    = 2.0
     @sample_rate = 22050
+    @total_frames = (@duration * @sample_rate).to_i
+    @cycles_per_frame = @frequency / @sample_rate
+    @sine_wave_cycle = 2 * Math::PI * @cycles_per_frame
+    @samples = samples_helper
     @note = MusicTheory::Note.new(frequency: @frequency, duration: @duration)
+    @note_2 = MusicTheory::Note.new(frequency: @frequency, duration: @duration, distort: true)
   end
 
   def test_total_frames
-    frames = (@duration * @sample_rate).to_i
-    assert_equal frames, @note.total_frames
+    assert_equal @total_frames, @note.total_frames
   end
 
   def test_cycles_per_frame
-    cycles_per_frame = @frequency / @sample_rate
-    assert_equal cycles_per_frame, @note.cycles_per_frame
+    assert_equal @cycles_per_frame, @note.cycles_per_frame
```

Comment:
By only having a single test per method, you're not testing the fact that you get different results with different inputs. Test as if an evil (or wildly incompetent) person will be the next person to maintain your code and they will have no problem with making your test pass by just plugging a single number into the method where before you were doing a calculation.